pkg/cmd/cli/describe/projectstatus_test.go

```diff
@@ -313,9 +312,9 @@ func TestProjectStatus(t *testing.T) {
 			ErrFn: func(err error) bool { return err == nil },
 			Contains: []string{
 				"In project example on server https://example.com:8443\n",
-				"svc/galera[default] (headless):3306",
+				"svc/galera (headless):3306",
```

Comment:
Here, situation is a bit more complicated. Fake clientset forces to always use namesapces, previously it was not required. I've added them and previously we've had parts of the petset created in default namespace and now it's all in one. Thus the change.